themes/console-home/layouts/partials/footer.html

```diff
@@ -10,16 +10,16 @@
                         <a class="menu-item menu-item-secondary" onclick="blur()" href="/betas/">Betas</a>
                     </li>
                     <li>
-                        <a class="menu-item menu-item-secondary" onclick="blur()" href="/interviews/">Interviews</a>
+                        <a class="menu-item menu-item-secondary" onclick="blur()" href="/jobs/">Jobs</a>
                     </li>
                     <li>
-                        <a class="menu-item menu-item-secondary" onclick="blur()" href="/articles/">Articles</a>
```

Comment:
@davidmytton `/articles/` is what I've replaced, in case it's not clear from the commit message.

Reply:
Ok with with